src/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ if(UNIX)  # LINUX, FREE_BSD, APPLE
     endif()
 
     list(APPEND PROJECT_NAMES pcm-sensor)
-    if(LINUX)
+    if(NOT APPLE)
```

Comment:
Hi @mizhka, thanks for you suggestion! 
I think it would look nicer if  2 "if (NOT APPLE)" cases will be combined together. 

hi @rdementi , 
do we have any restrictions which are not allowing build pcm-sensor-server on FreeBSD ?

Reply:
Hi @markovamaria , 

Thank you. Done.

Reply:
@markovamaria 
I am not aware of such restrictions